Sources/AloeStackView/Protocols/Highlightable.swift

```diff
@@ -13,28 +13,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import UIKit
+
 /**
- * Indicates that a row in an `AloeStackView` should change its background color to a
- * highlighted color when a user is touching it.
+ * Indicates that a row in an `AloeStackView` should be highlighted when it touches
  *
  * Rows that are added to an `AloeStackView` can conform to this protocol to have their
- * background color automatically change to a highlighted color when the user is pressing down on
+ * highlightAction(isHighlighting:) will be performed when the user is pressing down on
  * them.
  */
 public protocol Highlightable {
 
-  /// Checked when the user touches down on a row to determine whether the background color of the
-  /// row should be changed to a highlighted color.
+  /// If the user touches the row, the row should be highlighted.
   ///
   /// The default implementation of this method always returns `true`.
   var isHighlightable: Bool { get }
-
+  
+  /// You can custom highlight the touch action of each row in the AloeStackView
```

Comment:
Clarity: *Called when the highlighted state of the row changes.*

*Override this method to provide custom highlighting behavior for the row.*